src/smc-webapp/editor.coffee

```diff
@@ -1067,7 +1061,8 @@ class CodeMirrorEditor extends FileEditor
         cb?()
 
     codemirrors: () =>
-        return [@codemirror, @codemirror1]
+        c = [@codemirror, @codemirror1]
+        return underscore.filter(c, ((x) -> x?))
```

Comment:
Why do this when you can do: `(x for x in c when x?)`.  CoffeeScript has nice list comprehensions just like in Python (but with parens).

Also, it would be better to just define the list `[@codemirror, @codemirror1]` once, then change it once right when things are deleted, rather than filtering every single time.  

That said, this code is definitely all going to be deleted relatively soon, so I'm fine with it exactly as you wrote it.